Makefile

```diff
@@ -0,0 +1,46 @@
+.PHONY: uv-download
```

Comment:
using the action "astral-sh/setup-uv@v4" seems to be the preferred way:
https://docs.astral.sh/uv/guides/integration/github/#installation

Reply:
Was already done